### DISCUSSION TITLE
我想知道怎么取消按Alt键多选

### DISCUSSION THREAD

**Discussion by ioldyao** [Eugeny/tabby#7795] [Q&A]
每次我在Alt + Tab切换的时候，软件屏幕上鼠标就会显示十字的多选，很麻烦。我想知道怎么把Alt这个功能给取消了

**Comment by lawweiliang** [comment]
@ioldyao
English only please.
https://github.com/Eugeny/tabby/discussions/5533

![image](https://user-images.githubusercontent.com/34936929/228172727-cc58dcc4-bdf4-4085-844a-a25cf784c145.png)
